airflow/providers/postgres/hooks/postgres.py

```diff
@@ -149,7 +152,7 @@ def _serialize_cell(cell, conn):
         """
         return cell
 
-    def get_iam_token(self, conn):
+    def get_iam_token(self, conn: psycopg2.extensions.connection) -> Tuple[Any, Any, Union[int, Any]]:
```

Comment:
The return type here is too general, I think you can improve it more!

Reply:
Thanks man, I found the exactly type of return of each one variables

Reply:
@kenjihiraoka please can you tell me the exactly type of return of each one variables ?